### PR TITLE
change api "listnode" "nodeinfo" "metrics" path from "wallet" to "monitor"

### PR DESCRIPTION
**Pull request by Starsakary**
**What does this PR do?**

change api "listnode" "nodeinfo" "metrics" path from "wallet" to "monitor"

### CODE REVIEW

**Comment by chengtx01** [comment]
Old interfaces need to be compatible

File: framework/src/main/java/org/tron/core/services/http/solidity/SolidityNodeHttpApiService.java [hunk 1]
```diff
@@ -184,7 +184,7 @@ public void start() {
       context.addServlet(new ServletHolder(triggerConstantContractServlet),
           "/walletsolidity/triggerconstantcontract");
 
-      context.addServlet(new ServletHolder(getNodeInfoServlet), "/wallet/getnodeinfo");
+      context.addServlet(new ServletHolder(getNodeInfoServlet), "/monitor/getnodeinfo");
```
Comment (andelf) [inline framework/src/main/java/org/tron/core/services/http/solidity/SolidityNodeHttpApiService.java]: To be compared with the scope name `wallet`.
`getnodeinfo` may be under the scope of `node`, `system`, `network` or `status`.

> monitor: *n.* An instrument or device used for observing, checking, or keeping a continuous record of a process or quantity.

Comment (Starsakary) [reply framework/src/main/java/org/tron/core/services/http/solidity/SolidityNodeHttpApiService.java]: We are changing some apis to a new namespace, this is only a temporary name

File: framework/src/main/java/org/tron/core/services/http/solidity/SolidityNodeHttpApiService.java [hunk 2]
```diff
@@ -184,9 +184,11 @@ public void start() {
       context.addServlet(new ServletHolder(triggerConstantContractServlet),
           "/walletsolidity/triggerconstantcontract");
 
-      context.addServlet(new ServletHolder(getNodeInfoServlet), "/wallet/getnodeinfo");
+      context.addServlet(new ServletHolder(getNodeInfoServlet), "/walletsolidity/getnodeinfo");
```
Comment (chengtx01) [inline framework/src/main/java/org/tron/core/services/http/solidity/SolidityNodeHttpApiService.java]: Whether it is necessary to change ‘wallet’ to ‘walletsolidity’ needs to be considered again

**Pull request merged**